adafruit_pcf8591/analog_out.py

```diff
@@ -26,16 +26,21 @@
 
 * Author(s): Bryan Siepert
 """
+try:
+    import typing  # pylint: disable=unused-import
+    from adafruit_pcf8591.pcf8591.PCF8591 import PCF8591
+except ImportError:
+    pass
 
 
 class AnalogOut:
     """AnalogIn Mock Implementation for ADC Reads."""
 
-    def __init__(self, pcf, dac_pin=0):
+    def __init__(self, pcf: PCF8591, dac_pin: int = 0) -> None:
         """AnalogIn
 
         :param pcf: The pcf object.
-        :param ~digitalio.DigitalInOut DAC pin: Required pin must be P4
```

Comment:
I would clarify this as `adafruit_pcf8591.pcf8591.A0` since at first glance I thought it was talking about something like `board.A0`.  Sorry if I missed this on the first pass.

Reply:
make sure you review the way I did this to be correct

Reply:
Sorry, I wasn't very clear about that.  I would change this to `:param int dac_pin:`, and then change the text to read `Required pin must be ~adafruit_pcf8591.pcf8591.A0`.  Happy to explain if you're curious since you mentioned wanting to know more about Sphinx documentation!